_stbt/control.py

```diff
@@ -263,6 +265,123 @@ def _connect():
     return LircRemote(control_name, _connect)
 
 
+class RemoteFrameBuffer(object):
+    """Send a key-press to a set-top box running a VNC Remote Frame Buffer
+        protocol.
+        Expected key press input:
+            <KEY_LABEL>
+
+        control = RemoteFrameBuffer("192.168.0.123")
+        control.press("KEY_MENU")
+    """
+
+    # Map our recommended keynames (from linux input-event-codes.h) to the
+    # equivalent RFB keyname.
+    _KEYNAMES = {
+        'KEY_BACK': 0xE002,
+        'KEY_BLUE': 0xE203,
+        'KEY_CHANNELDOWN ': 0xE007,
+        'KEY_CHANNELUP': 0xE006,
+        'KEY_DOWN': 0xE101,
+        'KEY_ELPS': 0xEF00,
+        'KEY_FASTFORWARD': 0xE405,
+        'KEY_GREEN': 0xE201,
+        'KEY_GUIDE': 0xE00B,
+        'KEY_HELP': 0xE00A,
+        'KEY_HOME': 0xE015,
+        'KEY_INFO': 0xE00E,
+        'KEY_INPUTSELECT': 0xE010,
+        'KEY_INTERACT': 0xE008,
+        'KEY_0': 0xE300,
+        'KEY_1': 0xE301,
+        'KEY_2': 0xE302,
+        'KEY_3': 0xE303,
+        'KEY_4': 0xE304,
+        'KEY_5': 0xE305,
+        'KEY_6': 0xE306,
+        'KEY_7': 0xE307,
+        'KEY_8': 0xE308,
+        'KEY_9': 0xE309,
```

Comment:
What reference did you use to get these values?  I'm looking [keysymdef.h](https://cgit.freedesktop.org/xorg/proto/x11proto/plain/keysymdef.h) and I'm surprised that these codes don't seem to line up with `XK_0` (`0xffb0`) or `XK_KP_0` (`0x0030`).  Clearly I'm missing something here as these codes have been working for you.